src/compiler/types.ts

```diff
@@ -6145,6 +6145,7 @@ namespace ts {
         maxNodeModuleJsDepth?: number;
         module?: ModuleKind;
         moduleResolution?: ModuleResolutionKind;
+        moduleSuffixes?: string[];

```

Comment:
Also need test case when importing "foo.ts"

Reply:
When I add a case which does `import './foo.ts'`, I get an error:

```
TS2691: An import path cannot end with a '.ts' extension. Consider importing './foo' instead.
```

Reply:
I added a test which resolves a module to a .ts file in a node_modules package. Hopefully this is the case you were looking for.